mesos/src/main/scala/org/apache/spark/deploy/mesos/MesosClusterDispatcherArguments.scala

```diff
@@ -73,37 +94,55 @@ private[mesos] class MesosClusterDispatcherArguments(args: Array[String], conf:
       propertiesFile = value
       parse(tail)
 
+    case ("--conf") :: value :: tail =>
+      MesosClusterDispatcher.
+        parseSparkConfProperty(value, (k: String, v: String) => confProperties(k) = v)
+      parse(tail)
+
     case ("--help") :: tail =>
-      printUsageAndExit(0)
+        printUsageAndExit(0)
+
+    case ("--verbose") :: tail =>
+      verbose = true
+      parse(tail)
 
     case Nil =>
-      if (masterUrl == null) {
+      if (Option(masterUrl).isEmpty) {
         // scalastyle:off println
-        System.err.println("--master is required")
+        MesosClusterDispatcher.printStream.println("--master is required")
         // scalastyle:on println
         printUsageAndExit(1)
       }
 
-    case _ =>
+    case value =>
+      // scalastyle:off println
+      MesosClusterDispatcher.printStream.println(s"Unrecognized option: '${value.head}'")
```

Comment:
I guess it is noisy too here: https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

Counted 34 appearances of SparkSubmit. I just folloed the same idea I thought since it was previously reviewed I should do the same, guessing it was for clarity reasons.